nifi-api/src/main/java/org/apache/nifi/processor/AbstractProcessor.java

```diff
@@ -21,13 +21,19 @@
 public abstract class AbstractProcessor extends AbstractSessionFactoryProcessor {
 
     @Override
+    protected void init(ProcessorInitializationContext context) {
+        super.init(context);
```

Comment:
This seems like it is unnecessary.  What is this doing?  Even if a subclass implements their own init they'll need to call super anway, right?